truss/templates/control/control/helpers/truss_patch/model_code_patch_applier.py

```diff
@@ -14,16 +15,17 @@ def apply_model_code_patch(
     logger: logging.Logger,
 ):
     logger.debug(f"Applying model code patch {patch.to_dict()}")
-    action = patch.action
     filepath: Path = model_code_dir / patch.path
+    action = patch.action
```

Comment:
Why move this?

Reply:
no reason - I started a refactor but decided it would be better to take on in #388 , sorry for the confusion